src/index/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from '../core'
+export { default } from '../core'
```

Comment:
@huozhi Why is it necessary? This adds the named export for `useSWR`, doesn't it?

Reply:
Not sure you mean the file or the re-export of default.

For the index file, `./index/` matches `.` export, this is the name convention for bunchee, also acts like nodejs resolve behavior.

For the export, `default` need to export separatedly. we didn't export `useSWR` as named export in `core` or `index`

Reply:
Thank you! I misunderstood it